src/client/common/cy/events/show-tooltip.js

```diff
@@ -1,8 +1,9 @@
 
 const MetadataTip = require('../tooltips/');
 
-const bindShowTooltip = (cy) => {
-  cy.on('showTooltip', 'node', function (evt) {
+const bindShowTooltip = (cy,edgeTooltips) => {
```

Comment:
Minor nitpick but this should be renamed to communicate that this is a boolean.  It could be interpreted that the param edgeToolTips are actual tooltips being passed in.